packages/kbn-build/src/cli.js

```diff
@@ -31,11 +31,21 @@ function help() {
 }
 
 export async function run(argv) {
-  const options = getopts(argv, {
-    alias: {
-      h: 'help'
-    }
-  });
+  const options = parseArgv(argv);
+
+  // We can simplify this setup (and remove this extra handling) once Yarn
+  // starts forwarding the `--` directly to this script, see
+  // https://github.com/yarnpkg/yarn/blob/b2d3e1a8fe45ef376b716d597cc79b38702a9320/src/cli/index.js#L174-L182
+  if (hasDashDash(argv)) {
```

Comment:
I think `argv.includes('--')` is easier to read than a helper in another file... is there another reason we're doing this?

Reply:
Agreed, fixing